octobot/src/server/github_handler.rs

```diff
@@ -409,22 +455,24 @@ impl GithubEventHandler {
         participants.push(pull_request.user().clone());
         // add team members
         let teams = pull_request.teams();
-        let futures = teams
-            .iter()
-            .map(|t| self.github_session.get_team_members(&t.url));
-        let team_members = future::join_all(futures).await;
-        participants.extend(
-            team_members
-                .into_iter()
-                .filter_map(|r| match r {
-                    Ok(r) => Some(r),
+        for t in teams {
+            let team_members = self.team_members_cache.get(&t.slug);
+            if team_members != None {
+                participants.append(&mut team_members.unwrap().clone());
```

Comment:
```suggestion
            if let Some(mut members) != None {
                participants.extend(members.into_iter());
```

(i think this should work -- may have it slightly off)

more idomatic to extract with `if let Some(...)` and avoid `unwrap` if possible.

Reply:
Hmm..
![Screenshot 2023-05-29 at 7 56 57 PM](https://github.com/tanium/octobot/assets/4804434/054c4837-785d-471a-8905-3e1315c59464)

Reply:
oops. messed that up.

meant to say: `if let Some(team_members) = team_members`

example of syntax: https://github.com/tanium/octobot/blob/b1d10e680784111bff56de15b4e09d5e3f8e200d/octobot/src/server/github_handler.rs#L258

docs: https://doc.rust-lang.org/rust-by-example/flow_control/if_let.html

Reply:
Nice. Thanks!